RecoTauTag/RecoTau/plugins/RecoTauBuilderConePlugin.cc

```diff
@@ -122,7 +192,11 @@ RecoTauBuilderConePlugin::return_type RecoTauBuilderConePlugin::operator()(
 
   // Our tau builder - the true indicates to automatically copy gamma candidates
   // from the pizeros.
-  RecoTauConstructor tau(jet, getPFCands(), true);
+  double minAbsPhotonPt_inSignalCone=2.5, minRelPhotonPt_inSignalCone=0.1;//MB: sensible defults to set a DM hypothesis
```

Comment:
Yes, this changes default value to one which is used HPS algorithm. In the latter the value is set via a configuration option, but here are hardcoded the value because as in this way existing HLT configuration files work with updated producer and the value is not supposed to be modified.
This change does not affect HLT as in existing code variables depending on minRelPhotonPt_inSignalCone are not set and thus not used.

After said this: I tend to agree that minAbsPhotonPt_inSignalCone and minRelPhotonPt_inSignalCone should be set via configuration parameters.